PermissionsService/Models/PermissionResult.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using System.Net;
+
+namespace PermissionsService.Models
+{
+    public class PermissionResult
+    {
+        public List<ScopeInformation> Results
+        {
+            get; set;
+        }
+
+        public List<PermissionError> Errors
+        {
+            get; set;
+        }
+    }
+
+    public class PermissionError
+    {
+        public string Url
+        {
+            get; set;
+        }
+
+        public string Message
+        {
+            get; set;
+        }
```

Comment:
```suggestion
        } = string.Empty;
```

Should we decide to enable NRT for this project, we won't have to revisit this class this way